src/content/docs/en/guides/deploy/zerops.mdx

```diff
@@ -0,0 +1,178 @@
+
+  <PackageManagerTabs>
+    <Fragment slot="npm">
+    ```yaml
+      zerops:
+        - setup: hellothere
+          build:
+            base: nodejs@20
+            buildCommands:
+              - npm i
+              - npm run build
+            deploy:
+              - dist
+              - package.json
+              - node_modules
+            cache:
+              - node_modules
+              - package-lock.json
+          run:
+            start: node dist/server/entry.mjs
+          envVariables:
+            HOST: 0.0.0.0
+            NODE_ENV: production
+    ```
+    </Fragment>
+    <Fragment slot="pnpm">
+    ```yaml
+      zerops:
+        - setup: hellothere
+          build:
+            base: nodejs@20
+            buildCommands:
+              - pnpm i
+              - pnpm run build
+            deploy:
+              - dist
+              - package.json
+              - node_modules
+            cache:
+              - node_modules
+              - pnpm-lock.json
+          run:
+            start: node dist/server/entry.mjs
+          envVariables:
+            HOST: 0.0.0.0
+            NODE_ENV: production
+    ```
+    </Fragment>
+    <Fragment slot="yarn">
+    ```yaml
+      zerops:
+        - setup: astronode
+          build:
+            base: nodejs@20
+            buildCommands:
+              - yarn
+              - yarn build
+            deploy:
+              - dist
+              - package.json
+              - node_modules
+            cache:
+              - node_modules
+              - yarn.lock
+          run:
+            start: node dist/server/entry.mjs
+          envVariables:
+            HOST: 0.0.0.0
+            NODE_ENV: production
+    ```
+    </Fragment>
+  </PackageManagerTabs>
+
+### Trigger the pipeline by connecting the service to GitHub / GitLab
+To setup continuous deployment on either push to branch or a new release, go to your Node.js service detail and connect your Zerops service with either GitHub or GitLab repository.
+
+
+### Trigger the pipeline Using Zerops CLI (zcli)
+
+1. Install Zerops CLI
+```shell
+# To download the zcli binary directly,
+# use https://github.com/zeropsio/zcli/releases
+npm i -g @zerops/zcli
+```
+
+2. Open [`Settigs > Access Token Management`](https://app.zerops.io/settings/token-management) in Zerops app, generate a new access token
+
+3. Login
+```shell
+zcli login <token>
+```
+
+4. Navigate to your apps root (where zerops.yml is also placed)
+```shell
+zcli push
+```
+
+## Resources
+
+- [Deploying Astro to Zerops in 3 mins](https://medium.com/@arjun.js/how-to-deploy-astro-to-zerops-4230816a62b4)
+- [Detailed guide to create a Node.js service](https://docs.zerops.io/nodejs/how-to/create)
+- [Managing Environment Variables](https://docs.zerops.io/nodejs/how-to/env-variables)
+- [Updating Internal Ports](https://docs.zerops.io/nodejs/how-to/ports)
```

Comment:
```suggestion
- [Deploying Astro to Zerops in 3 mins](https://medium.com/@arjun.js/how-to-deploy-astro-to-zerops-4230816a62b4)
- [Detailed guide to creating a Zerops Node.js service](https://docs.zerops.io/nodejs/how-to/create)
```
We don't need to link to every little thing in the Zerops docs! the blog post and the creation guide are great, and then people will use Zerops docs to find whatever else they might need.